website/addons/dryad/model.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+from modularodm import fields
+from website.addons.base import AddonUserSettingsBase
+from website.citations.models import AddonCitationsNodeSettings
+from website.addons.dryad.serializer import DryadSerializer
+from website.addons.dryad.provider import DryadProvider
+
+
+class DryadUserSettings(AddonUserSettingsBase):
+    """
+     Stub in place for Dryad addon v2 upgrade.
+
+     As migrations are a little tricky, this stub is being left in since a user
+     settings class will be needed in the future.
+    """
+    oauth_provider = DryadProvider
+    serializer = DryadSerializer
+    has_auth = True
+
+
+class DryadNodeSettings(AddonCitationsNodeSettings):
+    """
+        A Dryad node is a collection of packages. Each package is specified by a DOI, and the title is saved automatically
+    """
+    provider_name = 'dryad'
+    oauth_provider = DryadProvider
+    serializer = DryadSerializer
+    dryad_package_doi = fields.StringField()
```

Comment:
@kwierman: the `validate_doi` validator doesn't like the `doi:` prefix you've been saving.  You'll need to strip that off and update the provider to prepend that before issuing your requests.